src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -3443,7 +3443,15 @@ private void GetContainerPropertiesInternal()
                     //
                     if (RuntimeId == Guid.Empty)
                     {
-                        ContainerObRoot = (string)computeSystemPropertiesType.GetProperty("ObRoot").GetValue(computeSystemPropertiesHandle);
+                        var pi = computeSystemPropertiesType.GetProperty("ObRoot");
```

Comment:
Please look a comment above for RuntimeId and follow the same pattern - check the field first.